LML.NPOManagement.Bll/Services/NotificationService.cs

```diff
@@ -224,17 +224,17 @@ public async void PasswordRecoverRequest(UserModel user)
 
         }
 
-        public async void EmailVerificationRequest(UserModel user)
+        public async void EmailVerificationRequest(UserModel user, string lang)
         {
             if (user == null)
             {
                 return;
             }
             var userInfo = user.UserInformations.FirstOrDefault();
 
-            var template = await GetTemplateByFileName("CheckingEmail.html");
+            var template = await GetTemplateByFileName("CheckingEmail.html", lang);
 
-            string token = TokenCreationHelper.GenerateJwtToken(user, _configuration, _userRepository);
+            string token = TokenCreationHelper.GenerateJwtToken(user, _configuration, _userRepository, 50);
 
```

Comment:
Here is another magic number. If this is a timeout, this has to be configured.

Reply:
last integer is used to set expiration time different than configuration, but it can be moved to configuration file